### DISCUSSION TITLE
Implementation of shift+F12 screenshot upload to osu like in stable.

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#16963] [Ideas] [answered]
Making a discussion since I couldn't find any others talking about this by searching upload screenshot.

I'd like to see this feature implemented hopefully soon and I don't think it'll take too long.
For those that don't know what I am talking about, in stable if you hold shift when taking a screenshot with F12 it will upload the screenshot to https://osu.ppy.sh/ss/ which allows you to easily share your screenshots of scores etc in chat with [https://osu.ppy.sh/ss/########/#### Text]
Very useful feature that I miss especially since we can't upload scores to leaderboards yet and I feel this would let us more easily share our lazer scores for the time being.

**Comment by TheFantasticLoki** [comment]
Also this doesn't need to be worked on immediately this is more of a bottom shelf request per say but I wanted to get a discussion made for it since I hadn't seen it mentioned anywhere before.

**Comment by Joehuu** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/10718

**Reply by TheFantasticLoki** [reply]
Thanks for pointing that out marking as answered
